GovUk.Frontend.AspNetCore.Extensions/Models/PaginationModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+
+namespace GovUk.Frontend.AspNetCore.Extensions.Models
+{
+    /// <summary>
+    /// Configuration settings for an instance of the GOV.UK Design System Pagination component.
+    /// </summary>
+    public class PaginationModel
+    {
+        /// <summary>
+        /// The current page.
+        /// </summary>
+        public int PageNumber { get; set; } = 1;
+
+        /// <summary>
+        /// The number of items per page.
+        /// </summary>
+        public int PageSize { get; set; } = 10;
+
+        /// <summary>
+        /// The total number of items to be paginated.
+        /// </summary>
+        public int TotalItems { get; set; }
```

Comment:
If `TotalItems` is a non-nullable property, should it not have a default value, like `PageSize` and `PageNumber`?

Reply:
That's the point though - it defaults to 0 which is the default I want. The other examples need a default value that's not the same as the default for the type. 

If you set a default value for something that is the default value for the type anyway, the compiler will warn you that it's an unnecessary thing to do.